NEWS

```diff
@@ -11,6 +11,333 @@
+* Fixed user memory handle support in rendezvous protocol
+* Fixed message split in rendezvous protocol to avoid having very small chunks
+* Improved performance estimations for protocols v2
+* Fixed receive descriptors leak in UCP AM rendezvous
+* Moved RKEY_PTR flag from memory domain to comnponent
+#### UCT
+* Fixed double free of server endpoint in TCP sockcm
+* Updated KNEM bandwidth to be dedicated resource rather than shared
+* Fixed race in CM when listener is destroyed during conn_req_cb invocation
+* Updated default bandwidth value for memory mapper transports
+* Disqualify posix transport if /dev/shm size is too small
+* Disqualify KNEM transport if memory registration fails with it
+#### RDMA CORE (IB, ROCE, etc.)
+* Fixed device error handling (prevent coredump when iface is down/up)
+* Multiple fixes in DC transport (error flows, flow control, etc)
+* Multiple fixes and cleanups in UD transport
+* Fixed MR registration (avoid atomic offset breaking region alignment)
+* Fixed indirect key registration (avoid creating atomic KSM on top of relaxed-order key)
+* Fixed thread domain usage for accelerated verbs transports
+* Added print of a particular syndrome on DEVX function failures
+* Fixed DEVX QP creation by setting proper ts_format attribute
+* Decreased size of DC endpoint
+* Fixed bandwidth calculation for RoCE LAGs
+* Fixed port counters setting for DEVX QPs
+* Fixed compile errors on SLES sp3
+* Removed errors during md open in case of strict memlock limit
+#### UCS
+* Removed async_max_events limit (e.g. to support many concurrent TCP connections)
+* Updated memory wc flush using DGH hint for ARM platform
+* Fixed deprecation warnings because of <sys/fcntl.h> includes
+* Added default bandwidth value for ZHAOXIN CPU
+* Fixed logger to correctly print a pid
+#### UCM
+* Fixed segfault in malloc when compiled with -flto
+* Fixed false positive memory leak detected by valgrind
+#### GPU (CUDA, ROCM)
+* Fixed ROCM memory type detection
+* Fixed ROCM IPC transport (use remote agent if available)
+* Fixed clang compilation errors in CUDA copy transport
+* Fixed ROCM memtype detection
+* Improved performance estimation of CUDA copy transport
+* Fixed send to self flows in ROCM
+#### Java
+* Disabled local link listener bindings
+#### Documentation
+* Updated GPU memory support section in FAQ
+* Aligned ucp_worker_fence() description with OSHMEM standard
+#### Tests
+* Fixed io_demo build on Ubuntu 20.04
+* Multiple improvements and fixes in UCP AM tests
+* Fixed connect_and_send_recv() in test_ucp_sockaddr
+* Fixed crash in test_ucp_sockaddr when RDMACM fails to resolve the peer address
+* Fixed UCT MD memory invalidation test (removed atomic access)
+* Multiple improvements/fixes to reduce overall testing time
+* Fixed gtest compilation error with -O1 optimization flag
+* Fixed memhooks test to load UCM first
+#### Tools
+* Replaced deprecated communication routines with *_nbx() APIs in ucp_hello_world
```

Comment:
this is Tests

Reply:
for 1.12 similar change is reported in `Core`. 
Also we mentioned `examples` in `Tools` section before

Reply:
it's definitely not Tools

Reply:
then, removed it, as Tests section contains just
`Multiple fixes and improvements in unit tests`